src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -138,11 +138,11 @@
         "instructor1OfCourse1":{
             "googleId":"idOfInstructor1OfCourse1.access",
             "courseId":"idOfTypicalCourse1.access",
-            "name":"Instructor1 Course1",
+            "name":"Instructor1 Course1</td></div>'\"",
```

Comment:
This polluting of test data is a problem. This is going to make the visual test output look like the UI is broken. I'd rather we do this either in a test that is clearly marked as a sanitization test or add html chars to only test object that is clearly marked as it is used for sanitization test. e.g. `Course with html chars </td>`. We want to avoid  the case where devs get desensitized to seeing weird html chars where they should not be and always thing 'oh, that must be a sanitization test' when in fact it is a different problem. So, lets minimize these weird test data and clearly mark them as sanitization tests.